source/version.js

```diff
@@ -20,7 +20,7 @@ class Version {
 	getNewVersionFrom(input) {
 		module.exports.validate(this.version);
 		if (!module.exports.isValidInput(input)) {
-			throw new Error(`Version should be either ${module.exports.SEMVER_INCREMENTS.join(', ')} or a valid semver version.`);
+			throw new Error(`Version should be either ${module.exports.SEMVER_INCREMENTS.join(', ')}, or a valid semver version.`);
```

Comment:
I added a comma here to match the comma that was in `prerequisite-tasks`:

https://github.com/sindresorhus/np/blob/aa23a92bcaf710068f01ed5fe50d3039b510e223/source/prerequisite-tasks.js#L62-L64